dev/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
             - 5432:5432
 
     solr6:
-        image: quay.io/alfresco/insight-engine:2.0.11
+        image: quay.io/alfresco/insight-engine:2.0.12-A1
```

Comment:
This should be bumped to alpha version? In comm-repo and ent-shara it was bumped to `2.0.12` GA